go.mod

```diff
@@ -1,9 +1,8 @@
 module github.com/edgexfoundry/go-mod-secrets
 
 require (
-	github.com/BurntSushi/toml v0.3.1
-	github.com/gorilla/context v1.1.1
-	github.com/gorilla/mux v1.7.1
+	github.com/edgexfoundry/go-mod-bootstrap v0.0.9
```

Comment:
What is the dependency on go-mod-bootstrap?  

go-mod-bootstrap [depends on go-mod-secrets](https://github.com/edgexfoundry/go-mod-bootstrap/blob/a952ca5a431b7efcbf4041c116e9a42cffebba07/go.mod#L7).  This creates a cyclical dependency.

Believe this can be removed -- I did so and tests still ran green.

Reply:
This was a mistake, and don't know how this change was made. Great catch